### ISSUE TITLE
Export Variants by Type

### ISSUE THREAD

**Issue by laststandjam** [kontent-ai/backup-manager-js#44]
### Motivation

Have run into an enviroment that is too large to export. Run into "This environment has more than 35000 variants. At this moment we do not support validation of environments of this size."

### Proposed solution

Expand the export filter to include specific Item Types or create a new filter that takes Item Type codenames and only exports items and variants of those types. 

`exportFilter: {
dataTypes: ["languageVariant"]
contentTypeFilter: ["type_code_name_1", "type_code_name_2"]
}`

### Additional context

My first thought was to target languages but assume this can't be done via Management API, as filtering is limited. However, MAPI does have this [endpoint](https://kontent.ai/learn/docs/apis/openapi/management-api-v2/#operation/list-language-variants-by-type) for types that could be leveraged to export large environments, or more granular control on what a user exports

**Comment by justinsmithintralox** [comment]
This issue was raised when trying to run a backup on Intralox.com. We have multiple languages and a decently-large site. A few months ago we were able to run backups, but after not using the utility for a few months, we get the following error when trying to run a backup:

```
data: {
      request_id: '76cc8b1c3ec6d09f',
      error_code: 10001,
      message: 'This environment has more than 35000 variants. At this moment we do not support validation of environments of this size.'
    }
```
Our config options for the backup include the following options:

```
  "action": "backup",
  "enableLog": true,
  "force": false,
  "baseUrl": null,
  "exportFilter": null
```

Happy to provide any additional information as needed.

**Comment by JiriLojda** [comment]
Hi @laststandjam,

Thank you for submitting the issue. We have just released a new tool that supports import/export. https://github.com/kontent-ai/data-ops We plan to deprecate backup manager in favour of this tool. Can you please re-test the scenario with that tool and submit an issue there if it persists?

**Comment by JiriLojda** [comment]
Hi @justinsmithintralox,

Thank you for the additional information. What version of the backup manager are you using? The project validation was removed in the version 4.2.0. From the error message you provided, it seems that the project validation is the problem. That would suggest you are using an older version. Unfortunately, the old validation call that was used before the 4.2.0 version doesn't support bigger projects, see https://kontent.ai/learn/docs/apis/openapi/management-api-v2/#operation/validate-content for more details.
Can you please try using our new data-ops tool https://github.com/kontent-ai/data-ops that I mentioned in the previous comment? As we are just deprecating the backup manager, switching to the newer tool would help you in the future. If you want to stick with the backup manager nevertheless, can you try version 4.2.0 or later to see if the problem persists?

**Comment by justinsmithintralox** [comment]
@JiriLojda - thanks for the info! We didn't realize that there was a new backup tool release - running with data-ops worked just fine.

There is one gotcha that we encountered that tripped us up a bit. We don't have access to the "roles" entity on our Kontent plan. Since that entity is included by default, we had to manually exclude it in order to successfully run a backup. Other than that, everything worked great! Thanks for the reply.
